### PR TITLE
portaudio 19.6.0

### PR DESCRIPTION
**Pull request by equal-l2**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Also remove universal binary support since audit says it's deprecated.

### CODE REVIEW

**Comment by equal-l2** [comment]
Well, should the version be `19.20161030`?

**Comment by ilovezfs** [comment]
https://app.assembla.com/wiki/show/portaudio/ReleaseNotes

They at least refer to it as 19.6.0 in the release notes.

**Comment by equal-l2** [comment]
@ilovezfs 
>They at least refer to it as 19.6.0 in the release notes.

Yes, but audit says
> `stable version should not decrease (from 19.20140130 to 19.6.0)`

**Comment by ilovezfs** [comment]
Then yes we'll either need to also add `version_scheme 1` or go with `version "19.20161030"`.

**Comment by equal-l2** [comment]
The upstream seems to go with `x.y.z` version scheme from now on.
https://lists.columbia.edu/pipermail/portaudio/2016-November/000937.html

**Comment by ilovezfs** [comment]
Cool. Thanks for finding that message.

File: Formula/portaudio.rb [hunk 1]
```diff
@@ -15,16 +17,13 @@ class Portaudio < Formula
...
-
   depends_on "pkg-config" => :build
 
   def install
-    ENV.universal_binary if build.universal?
     system "./configure", "--prefix=#{prefix}",
                           "--disable-debug",
                           "--disable-dependency-tracking",
-                          "--enable-mac-universal=#{build.universal? ? "yes" : "no"}",
+                          "--enable-mac-universal=no",
```
Comment (ilovezfs) [inline Formula/portaudio.rb]: is this already the default?

Comment (equal-l2) [inline Formula/portaudio.rb]: No, still needed to build non-universal libraries.

Comment (ilovezfs) [inline Formula/portaudio.rb]: yikes. OK thanks for checking

**Comment by ilovezfs** [comment]
@equal-l2 could I sweet talk you into adding a test block?

**Comment by equal-l2** [comment]
@ilovezfs I wrote a simple test. Is this enough?
(I'm not so familiar with portaudio to write a comprehensive test)

**Comment by ilovezfs** [comment]
@equal-l2 yup that looks good. You can tweak it so that it works while unlinked and for head too, by making it:
```
    system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-lportaudio", "-o", "test"
    system ENV.cxx, "test.cpp", "-I#{include}", "-L#{lib}", "-lportaudiocpp", "-o", "test_cpp"
    assert_match stable.version.to_s, shell_output("./test")
    assert_match stable.version.to_s, shell_output("./test_cpp")
```

**Review by ilovezfs — APPROVED** [review]

**Comment by nijikon** [comment]
@BrewTestBot test this please

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Pull request closed without merging**